explorer/app/components/Header/Header.styles.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { Theme } from "@mui/material";
+
+export const HEADER_HEIGHT = 56;
+
+const hideOnMobile = ({ theme }: { theme: Theme }) => css`
+  ${theme.breakpoints.down("lg")} {
+    ${css`
+      display: none;
+    `.styles}
+  }
+`;
+
+export const Container = styled.div`
+  display: flex;
```

Comment:
For consistency with other projects would you mind ordering all css alphabetically (as much as technically is possible) 🙏🏽

Reply:
Sure thing!